frontend/src/containers/RegistrationContainer/RegistrationContainer.js

```diff
@@ -0,0 +1,62 @@
+import { useState } from "react";
+import Registration from "../../components/Registration/Registration";
+import endpoints from "../../properties/endpoints";
+import { postToAPI } from "../../util/network";
+import alertTypes from "../../properties/alerttypes";
+import Alert from "../../components/Alert/Alert";
+
+function formatDetails(registrationDetails) {
+	const MONTH_ADJUSTER = 1;
```

Comment:
A bit unclear. I understand that it's adjusting month, but it's still just a magic number now.